src/configure/configure.ts

```diff
@@ -116,6 +116,69 @@ class Orchestrator {
         }
     }
 
+    private async getAzureResource() {
+        var azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(this.inputs.pipelineConfiguration);
+        var resource = null;
+        try {
+            resource = await azureResourceSelector.getAzureResource(this.inputs);
+            return resource;
+        }
+        catch (err) {
+            if (err.message === Messages.setupAlreadyConfigured) {
+                this.continueOrchestration = false;
+                return;
+            }
+            else {
+                throw err;
+            }
+        }
+    }
+
+    private async selectTemplate(resource: GenericResource, inputs: WizardInputs) {
+        switch (resource.type) {
+            case TargetResourceType.AKS:
+                let templateParameterHelper = new TemplateParameterHelper();
+                let template = this.inputs.potentialTemplates[0] as LocalPipelineTemplate;
+                inputs.pipelineConfiguration.template = this.inputs.potentialTemplates[0];
+                await templateParameterHelper.setParameters(template.parameters, inputs);
+                break;
+
+            case TargetResourceType.WebApp:
+                var shortlistedTemplates = [];
+                shortlistedTemplates = this.inputs.potentialTemplates.filter((template) => template.targetKind === resource.kind);
+                if (shortlistedTemplates.length > 1) {
+                    inputs.pipelineConfiguration.template = shortlistedTemplates.find((template) => template.templateType === TemplateType.remote);
```

Comment:
this can be undefined if no remote template is found. In such case may be use first element. Please handle such corner cases.

Reply:
Such case would occur if we have multiple local templates for the same targetKind and don't have any in the remote template. Please help me a bit in understanding this possibility.

Reply:
The possibility is almost zero. But none the less it is good idea to handle such corner cases explicitly and log appropriate error messages. Otherwise the error message would be too cryptic to understand the root cause without debugging.

Reply:
Added the required check